content/4_Gold/LIS.mdx

```diff
@@ -92,10 +92,7 @@ public static int findLis(int[] a) {
 <PySection>
 
 ```py
-from typing import List
-
-
-def find_lis(a: List[int]) -> int:
+def find_lis(a: list[int]) -> int:
```

Comment:
this doesn't compile in python 3.8 and lower (https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc) so maybe just leave it as is? note that CF currently uses python 3.8 and USACO uses python 3.6

Reply:
alr, everything should be back to using `typing` now